src/material/slider/slider-thumb.html

```diff
@@ -1,10 +1,7 @@
-<div class="mdc-slider__value-indicator-container" *ngIf="discrete" #valueIndicatorContainer>
+<div class="mdc-slider__value-indicator-container" [ngClass]="{'mdc-slider__thumb--with-indicator': _isValueIndicatorVisible}" *ngIf="discrete" #valueIndicatorContainer>
```

Comment:
I think `ngClass` adds unnecessary bytes, can you just do `[class.mdc-slider__thumb--with-indicator]="_isValueIndicatorVisible"`